packages/central-server/src/apiV2/surveyResponse.js

```diff
@@ -135,6 +136,11 @@ export async function surveyResponse(req, res) {
     );
 
     results = await saveResponsesToDatabase(transactingModels, userId, responses);
+
+    if (req.query.waitForAnalyticsRebuild) {
```

Comment:
I don't necessarily love adding these flags, however they're required for PSSS. And until we have a cleaner system for ingesting survey responses I think they'll stay.